docs/src/main/paradox/routing-dsl/exception-handling.md

```diff
@@ -79,6 +79,13 @@ To understand the performance implications of (mis-)using exceptions,
 have a read at this excellent post by A. Shipilёv: [The Exceptional Performance of Lil' Exception](https://shipilev.net/blog/2014/exceptional-performance).
 @@@
 
+
+@@@ note
+Please note that since version `10.1.6`, the default `ExceptionHandler` will also discard the entity bytes automatically. If you want to change this behavior,
+please refer to @ref[the section above](exception-handling.md#exception-handling) if you want to change this behavior; however, might cause connections to stall
```

Comment:
two times: "if you want to change this behavior"

Reply:
Thanks!